cmd/operator-sdk/olmcatalog/gen-csv.go

```diff
@@ -54,10 +56,14 @@ Configure CSV generation by writing a config file 'deploy/olm-catalog/csv-config
 	}
 
 	genCSVCmd.Flags().StringVar(&csvVersion, "csv-version", "", "Semantic version of the CSV")
-	genCSVCmd.MarkFlagRequired("csv-version")
+	if err := genCSVCmd.MarkFlagRequired("csv-version"); err != nil {
+		log.Fatalf("Failed to mark `csv-version` flag for `olm-catalog gen-csv` subcommand as required")
```

Comment:
```suggestion
		log.Fatalf("Failed to mark `csv-version` flag for `olm-catalog gen-csv` subcommand as required: %v", err)
```
Or acutally why not 

Just realized we're using `Fatalf()` without formatting in the error in a bunch of other places as well.
https://github.com/operator-framework/operator-sdk/blob/master/cmd/operator-sdk/add/controller.go#L56
https://github.com/operator-framework/operator-sdk/blob/master/cmd/operator-sdk/add/controller.go#L60
https://github.com/operator-framework/operator-sdk/blob/master/cmd/operator-sdk/add/crd.go#L48
https://github.com/operator-framework/operator-sdk/blob/master/cmd/operator-sdk/add/crd.go#L52

But we can fix those outside of this PR.